packages/ui/Input/Select.tsx

```diff
@@ -261,9 +265,7 @@ interface SelectMenuItemProps<T> {
 function SelectMenuItem<T>({ option, onClick, className }: SelectMenuItemProps<T>) {
   const { open, type, handleOptionClick, selected, multiple } = useSelectContext<T>();
 
-  const isSelected = multiple
-    ? selected?.some((item) => item.value === option.value)
-    : selected?.[0]?.value === option.value;
+  const isSelected = Boolean(selected?.find((item) => item.value === option.value));
```

Comment:
option.value랑 같은 것이 있는지를 체크하는 맥락이라면 아래가 좀 더 간결할 것 같아요.

```suggestion
  const isSelected = selected?.some((item) => item.value === option.value);
```